### ISSUE TITLE
headphone jack not detected

### ISSUE THREAD

**Issue by howdev** [linuxmint/Cinnamon#7663]
```
 * Cinnamon version  3.6.7
 * Distribution - Mint 18.3
 * Graphics hardware (not related to graphics)
 * 64bit
 ```

**Issue**
Front headphone not detected when plug in. Intel HD ALC1150
Line-Out is detected.

**Steps to reproduce**
plug in headphone to front headphone jack

**Expected behaviour**
headphone should show in pulseaudio output devices

**Other information**

**Comment by icarter09** [comment]
@howdev have you upgraded to the latest Cinnamon version? And if so, does this issue still persist?

**Comment by howdev** [comment]
@icarter09 I have upgraded to Mint 19 with new Kernel 4.15 and Cinnamon 3.8. Still does not detect front headphone jack.

**Comment by icarter09** [comment]
@howdev does the Sound applet pick up on the headphones? I think this might be related to #8016.

**Comment by Vahan86** [comment]
@howdev, is this still an issue in Cinnamon 4.2?

**Comment by howdev** [comment]
@Vahan86 Yes still an issue in Mint 19.2 Cinnamon 4.2

**Comment by icarter09** [comment]
@howdev are you using any other frameworks for handling sound? I'm not able to reproduce this issue using the steps provided.

**Comment by howdev** [comment]
@icarter09 No, I am not using any other frameworks for sound, just the default. 
the front headphone still don't work. The line-out works, the front mic works.

**Comment by howdev** [comment]
it looks like is a sound system problem with certian sound hardware. The one I got is ALC1150 Intel

When you use hdajackretask and make the front port headphone, the headphone will be detected when plug and line out will auto muted. Then comes another problem, there is noise coming from the line-out with automute. When you disable automute the noise is gone. but sound comes both headphone and line-out. 

The pulseaudio settings shows the line-out is replaced by the headphone if the headphone is plug in. It is not separate device.  If separate device then automute can be disabled, and output device is auto switched
